Source/WebCore/html/HTMLElement.h

```diff
@@ -159,6 +159,8 @@ class HTMLElement : public StyledElement {
     void setPopover(const AtomString& value) { setAttributeWithoutSynchronization(HTMLNames::popoverAttr, value); };
     void popoverAttributeChanged(const AtomString& value);
 
+    virtual void handleInvokeInternal(const AtomString&) {}
```

Comment:
```suggestion
    virtual void handleInvokeInternal(const AtomString&) { }
```
This needs a space to make the style bot happy.